pyvista/core/pointset.py

```diff
@@ -122,6 +128,175 @@ def remove_cells(self, ind, inplace=True):
         target.RemoveGhostCells()
         return target
 
+    def points_to_double(self):
+        """Convert the points datatype to double precision.
+
+        Note
+        ----
+        This operates in place
+
+        Examples
+        --------
+        Create a mesh that has points of the type ``float32`` and
+        convert the points to ``float64``.
+
+        >>> import pyvista
+        >>> mesh = pyvista.Sphere()
+        >>> mesh.points.dtype
+        dtype('float32')
+        >>> _ = mesh.points_to_double()
+        >>> mesh.points.dtype
+        dtype('float64')
+
+        """
+        if self.points.dtype != np.double:
+            self.points = self.points.astype(np.double)
+        return self
+
+    def translate(self, xyz: Union[list, tuple, np.ndarray], transform_all_input_vectors=False, inplace=None):
+        """Translate the mesh.
+
+        To Do
+        -----
+        `transform_all_input_vectors` is not handled when modifying inplace
+
+        Parameters
+        ----------
+        xyz : list or tuple or np.ndarray
+            Length 3 list, tuple or array.
+
+        Examples
+        --------
+        Create a sphere and translate it by ``(2, 1, 2)``.
+
+        >>> import pyvista
+        >>> mesh = pyvista.Sphere()
+        >>> mesh.center
+        [0.0, 0.0, 0.0]
+        >>> trans = mesh.translate((2, 1, 2), inplace=True)
+        >>> trans.center
+        [2.0, 1.0, 2.0]
+
+        """
+        if inplace is None:
+            # Raise a warning about inplace changing
+            warnings.warn(DEFAULT_INPLACE_WARNING)
```

Comment:
Other deprecations have used `PyvistaDeprecationWarning`: https://github.com/pyvista/pyvista/blob/5d1e19366d3121a2a41ba163064064e235f0f604/pyvista/utilities/features.py#L104-L108

Reply:
Thanks for pointing this out, I wasn't aware of that!